rnafold/layers/feature_preprocessing.py

```diff
@@ -17,12 +17,12 @@ def __init__(self, c_m, c_z, sec_struct_bins=20, splash_bins=5, shape_bins=10):
         self.shape_binner = Binner(num_bins=shape_bins + 1, bin_size=0.15, first_bin_upper_bound=-0.1)
         self.splash_binner = Binner(num_bins=splash_bins, bin_size=1. / splash_bins)
 
-        self.embedding_a, self.embedding_b = nn.Embedding(NUM_RESIDUE_TOKENS, c_z), nn.Embedding(NUM_RESIDUE_TOKENS, c_z)
+        self.embedding_a, self.embedding_b = nn.Embedding(NUM_RESIDUE_VOCAB, c_z, padding_idx=0), nn.Embedding(NUM_RESIDUE_VOCAB, c_z, padding_idx=0)
```

Comment:
Just in case padding index isn't zero:
```suggestion
        self.embedding_a, self.embedding_b = nn.Embedding(NUM_RESIDUE_VOCAB, c_z, padding_idx=RESIDUE_VOCAB_ORDER[PADDING_TOKEN]), nn.Embedding(NUM_RESIDUE_VOCAB, c_z, padding_idx=RESIDUE_VOCAB_ORDER[PADDING_TOKEN])
```